docs/proto/provider.md

```
@@ -1137,7 +1137,7 @@ specification for bid eligibility against the provider's capabilities.
  | ----- | ---- | ----- | ----------- |
  | `passed` | [bool](#bool) |  | Passed indicates whether the deployment group passed bid screening. |
  | `reasons` | [string](#string) | repeated | Reasons contains explanations when screening fails. |
- | `resource_offer` | [akash.market.v1beta5.ResourceOffer](#akash.market.v1beta5.ResourceOffer) |  | ResourceOffer is the provider's resource offer for the screened group, if passed. |
+ | `resource_offers` | [akash.market.v1beta5.ResourceOffer](#akash.market.v1beta5.ResourceOffer) | repeated | ResourceOffers contains the provider's resource offers for the screened group, one per service in the GroupSpec. |
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)